.github/workflows/build_and_test.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         with:
           path: ~/.konan
           key: ${{ runner.os }}-gradle-konan-${{ env.KOTLIN_VERSION }}
-      - name: Build all (excluding save-orchestrator-common, save-orchestrator and save-sandbox)
+      - name: Build all (excluding save-orchestrator-common, save-orchestrator, save-sandbox, save-backend and save-frontend)
```

Comment:
may be now it's easy to say what do we build exactly, instead of `all excluding a,b,c,d,e,f,g,...` 😁

Reply:
but still, it doesn't help. I see now only that frontend fails

Reply:
refactored to build some specific modules at first and left at the end